cobblerclient.go

```diff
@@ -102,6 +98,29 @@ func (c *Client) Call(method string, args ...interface{}) (interface{}, error) {
 	return result, nil
 }
 
+func (c *Client) setCachedVersion() error {
+	if c.CachedVersion != (CobblerVersion{}) {
+		return nil
+	}
```

Comment:
I don't get your point here sadly. If the cached version is already set we don't need to set it again. If you want to set the version again you have to manually invalidate the cache via the provided method.

Reply:
I know this is an edgecase, but are there not situations where you update cobbler and the version should be updated? Or do we automatically invalidate the cache in this case?

Reply:
This won't work anyway since the cached token will be invalid. But if we assume that this corner case is correct then we cannot cache the version at all since we need to check before any request if the version has changed.